html/introduction-to-html/tasks/basic-text/basic-text3-download.html

```diff
@@ -27,9 +27,14 @@
       }
 
       p, em {
-        color: gray;
         margin: 0.5em 0;
       }
+      p {
```

Comment:
I'd remove this rule; there is no point setting the paragraph color to black, as that is the browser default anyway.